packages/metal-quartz-link/src/MetalQuartzLink.soy

```diff
@@ -0,0 +1,56 @@
+{namespace MetalQuartzLink}
+
+/**
+ * This renders an anchor element.
+ */
+{template .render}
+	{@param? dataToggle: string}
+	{@param? elementClasses: string}
+	{@param? href: string}
+	{@param? html: html}
+	{@param? icon: ?}
+	{@param? label: html}
+	{@param? id: string}
+
+	{if $html}
+		{$html}
```

Comment:
If someone is passing in their own `html`, it seems like they might as well not use the component. Do you think we need this?